src/bin/observable.ts

```diff
@@ -144,7 +152,8 @@ try {
           config: await readConfig(config, root),
           message,
           ifBuildMissing: (ifMissing ?? "prompt") as "prompt" | "build" | "cancel",
-          ifBuildStale: (ifStale ?? "prompt") as "prompt" | "build" | "cancel" | "deploy"
+          ifBuildStale: (ifStale ?? "prompt") as "prompt" | "build" | "cancel" | "deploy",
+          force: build ? "build" : noBuild ? "deploy" : null
```

Comment:
We need to use the tokens parsing like we do for `--no-open` so that `--build` and `--no-build` have equal precedence and instead it is order which determines what happens. I’ll push a commit to implement this.

We might also error if both `--build` or `--no-build` is specified and `--if-missing` or `--if-stale` is specified, or log a warning saying that `--if-missing` or `--if-stale` is ignored if `--build` or --no-build` is specified? And maybe even log a deprecation notice for `--if-missing` and `--if-stale` entirely?

Reply:
I’ve done the first part in 83c07ca.

Reply:
And the second part is addressed in 0859004.